powerline/config_files/colorschemes/vim/__main__.json

```diff
@@ -7,6 +7,7 @@
 		"file_encoding":      "file_format",
 		"file_type":          "file_format",
 		"branch":             "information:additional",
+		"stash":              "information:additional",
```

Comment:
I did understand the string assignments in color schemes as categories instead of aliases. Looks like they provide exactly what I intended.

The *default* color scheme defines `branch_dirty` and `branch:divider` in the global scheme, so I aliased it there. In *solarized*, `stash:divider` is only defined in the definition for Vim, so declared the alias there, too.